### DISCUSSION TITLE
Control over CSS & JS minification settings?

### DISCUSSION THREAD

**Discussion by ryanemitchell** [webdiscus/html-bundler-webpack-plugin#30] [Q&A] [answered]
Is it possible to easily customize the minification and optimization settings for JavaScript and CSS in a production build? I'm looking to disable CssMinimizer and/or Terser, or to be able to customize their settings depending on the specific needs of the project. Can this customization be achieved through the configuration in the HtmlBundlerPlugin instance?

I have a full configuration running, but it lacks control over minification. (html is working just fine from[ these docs](https://github.com/webdiscus/html-bundler-webpack-plugin#option-minify-options).) I've been working with the [QuickStart](https://github.com/webdiscus/html-bundler-webpack-plugin#install) configuration to try to get it working. 

I'm stuck and I appreciate the help, thanks!

**Comment by webdiscus** [comment, marked as answer]
Hello @ryanemitchell,

the plugin provides no options for CSS & JS minification.

1. in production mode, JS will be minified automatically, no additional settings required
2. the plugin avoid extracting *LICENSE.txt files, no settings in Terser required
3. in mostly use cases used SASS with `sass-loader` which minified CSS in production mode automatically, no CssMinimizer required.

If you use pure CSS without `sass-loader`, then you should use the CssMinimizer to minify CSS. This is out of the plugin's control.

**Reply by ryanemitchell** [reply]
Got it. Thank you @webdiscus!
